packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -275,8 +275,8 @@ export const CopyAlternateIcon = (
 );
 
 export const StarIcon = (
-  <svg viewBox="0 0 14 13" fill="none">
-    <path d="M6.81109 1.06634C6.87335 0.887223 7.12665 0.887225 7.18891 1.06634L8.2487 4.11521C8.41349 4.5893 8.85592 4.91074 9.35773 4.92097L12.5849 4.98673C12.7745 4.99059 12.8527 5.2315 12.7016 5.34606L10.1295 7.29614C9.72951 7.59936 9.56052 8.11947 9.70586 8.59987L10.6406 11.6894C10.6955 11.8709 10.4905 12.0198 10.3349 11.9115L7.68542 10.0678C7.27344 9.78113 6.72656 9.78113 6.31458 10.0678L3.66511 11.9115C3.50946 12.0198 3.30454 11.8709 3.35945 11.6894L4.29414 8.59988C4.43948 8.11947 4.27049 7.59936 3.87053 7.29614L1.29837 5.34606C1.14727 5.2315 1.22554 4.99059 1.41513 4.98673L4.64227 4.92097C5.14408 4.91074 5.58651 4.5893 5.7513 4.11521L6.81109 1.06634Z" />
+  <svg viewBox="0 0 44 42" fill="none">
+    <path d="M20.7844 2.02721C21.0224 1.43432 21.1415 1.13788 21.2686 1.00154C21.664 0.577482 22.336 0.577482 22.7314 1.00154C22.8585 1.13788 22.9776 1.43432 23.2156 2.02721L27.9005 13.6936C27.9705 13.8679 28.0055 13.955 28.0515 14.029C28.1892 14.2509 28.4083 14.4101 28.6619 14.4725C28.7465 14.4934 28.8402 14.4997 29.0276 14.5124L41.5706 15.3629C42.2081 15.4061 42.5268 15.4277 42.6958 15.5065C43.2213 15.7515 43.4289 16.3906 43.1478 16.8977C43.0574 17.0607 42.8123 17.2655 42.322 17.6752L32.6743 25.7359C32.5302 25.8563 32.4581 25.9165 32.4019 25.9831C32.2335 26.1827 32.1498 26.4402 32.1687 26.7007C32.1751 26.7876 32.198 26.8787 32.2438 27.0608L35.311 39.2528C35.4668 39.8724 35.5448 40.1822 35.5221 40.3672C35.4515 40.9427 34.9078 41.3377 34.3387 41.227C34.1557 41.1915 33.8851 41.0216 33.344 40.6819L22.6965 33.9973C22.5375 33.8974 22.4579 33.8475 22.3773 33.8146C22.1354 33.7161 21.8646 33.7161 21.6227 33.8146C21.5421 33.8475 21.4625 33.8974 21.3035 33.9973L10.656 40.6819C10.1149 41.0216 9.84432 41.1915 9.66134 41.227C9.0922 41.3377 8.54855 40.9427 8.47792 40.3672C8.45522 40.1822 8.53315 39.8724 8.68903 39.2528L11.7562 27.0608C11.802 26.8787 11.8249 26.7876 11.8313 26.7007C11.8502 26.4402 11.7665 26.1827 11.5981 25.9831C11.5419 25.9165 11.4698 25.8563 11.3257 25.7359L1.67804 17.6752C1.18774 17.2655 0.94259 17.0607 0.852212 16.8977C0.57109 16.3906 0.778749 15.7515 1.30424 15.5065C1.47318 15.4277 1.7919 15.4061 2.42935 15.3629L14.9724 14.5124C15.1598 14.4997 15.2535 14.4934 15.3381 14.4725C15.5917 14.4101 15.8108 14.2509 15.9485 14.029C15.9945 13.955 16.0295 13.8679 16.0995 13.6936L20.7844 2.02721Z" fill="#D1D1D9"/>
```

Comment:
gotta remove this fill color and we need to make sure this icon is being properly used on all themes and locations (I believe it's used for favorites)